docs/en/observability/synthetics-intro.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[monitor-uptime-synthetics]]
+= Synthetic monitoring
+
+++++
+<titleabbrev>Synthetics (beta)</titleabbrev>
+++++
+
+beta[]
+
+// to do
+++++
+<div style="background-color:#feedf5;color:#F04E98;height:100px;width:100%;text-align:center;padding-top:35px;margin-top:32px;margin-bottom:32px">
+  To do: Add a note on the relationship between "Uptime" and "Synthetics (beta)"
```

Comment:
Here’s a starting point for some copy for explaining the difference between Uptime and Synthetics (I think we can add links in the copy to other docs too, but have left these out for now).

@drewpost and @florent-leborgne I’d be keen to see if you have any input here too.

> Uptime is the Observability Kibana app for viewing result data from lightweight monitors running through Heartbeat configured with the traditional `heartbeat.yml` file. This is for TCP, HTTP or ICMP monitors that you have configured and run from your own infrastructure with Heartbeat natively. If you are using browser based monitors, you should move to using the Synthetics app.
> 
> The Synthetics Observability Kibana app offers a richer management and reporting experience, particularly for browser based monitors as well as for lightweight monitors. The Synthetics app is tailored for monitors that are running from Elastic’s global managed testing infrastructure (requiring no platform set up or maintenance on your part) or from Private Locations. These monitors can be managed (created, edited, deleted etc.) directly from within the Synthetics UI, or managed externally (such as via automation, or CI/CD pipelines) using our Project Monitor feature. The Synthetics app also offers a far richer workflow with more capabilities such as triaging and responding to alerts.